addons/survey/views/survey_question_views.xml

```diff
@@ -30,8 +30,8 @@
                         <group>
                             <field name="question_type" widget="radio" attrs="{'required': [('is_page', '=', False)]}" />
                         </group>
-                        <group>
-                            <div class="mx-lg-auto w-lg-50 d-none d-sm-block o_preview_questions">
+                        <div>
+                            <div class="mx-lg-auto w-lg-50 d-none d-lg-block w-100 o_preview_questions">
```

Comment:
This is weird, `w-100` is never used as it is `d-none` up to size `lg` and `w-lg-50`. What did you want to display on sm/md viewports?